bot.py

```diff
@@ -48,17 +51,22 @@ class MusicBot(discord.Client):
     # pylint: disable=too-many-instance-attributes
 
     COMMAND_PREFIX = "!"
-    guild = None
-    handlers = None
-    voice_client = None
-    song_queue = None
-    current = None
-    last_text_channel = None
-
-    def __init__(self, guild):
+
```

Comment:
Here guild, handlers etc look like they are static variables, but they are only used as member variables in the code. So I think it is confusing to define them this way since we don't use them as static variables and therefore should just initialize them in the constructor.

Reply:
Ignore me, see [SE](https://softwareengineering.stackexchange.com/q/254576)